docs/jailer.md

```diff
@@ -280,10 +280,13 @@ Note: default value for `<api-sock>` is `/run/firecracker.socket`.
 ### Known limitations
 
 - When passing the --daemonize option to Firecracker without the --new-ns-pid
-  option, the Firecracker process will have a different pid than the Jailer
-  process. The suggested workaround to get Firecracker process's pid in this
-  case is using `--new-pid-ns` flag and read Firecracker's pid from the
-  `firecracker.pid` file present in the jailer's root directory.
+  option, the Firecracker process will have a different PID than the Jailer
+  process. As a workaround to get Firecracker PID, the Jailer now stores the PID
```

Comment:
Using "now" sounds CHANGELOG to me (like "we didn't do A, but now we do"). This is a doc and it should always the current behavior.
```suggestion
  process. As a workaround to get Firecracker PID, the Jailer stores the PID
```